fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can we handle this in a separate issue, please? I think as it is proposed here, it's already an improvement.